FEXCore/Source/Utils/FutexSpinWait.cpp

```diff
@@ -0,0 +1,39 @@
+#include "Utils/FutexSpinWait.h"
+
+namespace FEXCore::Utils {
+#ifdef _M_ARM_64
+  std::atomic<bool> FEXCore::Utils::FutexSpinWait::StaticDataInitialized;
+  std::mutex FEXCore::Utils::FutexSpinWait::StaticInitMutex;
+  uint32_t FEXCore::Utils::FutexSpinWait::CycleCounterFrequency;
+  uint64_t FEXCore::Utils::FutexSpinWait::CyclesPerNanosecond;
```

Comment:
Why not initialize these directly?

```suggestion
  uint32_t FEXCore::Utils::FutexSpinWait::CycleCounterFrequency = GetCycleCounterFrequency();
  uint64_t FEXCore::Utils::FutexSpinWait::CyclesPerNanosecond = CalculateCyclesPerNanosecond();
```

In fact, doing so allows for dropping CalculateCyclesPerNanosecond entirely and just moving its comment up here. Then you don't need the `Fairly trivial calculation but broken out for additional information.` disclaimer either, which was a bit of a bumpy introduction anyway.